### ISSUE TITLE
Is there any command in numpy to find eigen value and eigen vectors similar to Julia's Eigen without scaling the matrix? 

### ISSUE THREAD

**Issue by Vishnu213** [numpy/numpy#21969]
This is the command in Julia with the scale option:
D,V=eigen(K; permute=true ,scale=false,sortby=nothing).

Also in Matlab, we can carry out the similar operation using: eig(K, 'nobalance') command.

Thank you in advance!

**Comment by MuhammadNizamani** [comment]
Yes, NumPy provides a function numpy.linalg.eig() to compute eigenvalues and eigenvectors of a square array. By default, it computes the eigenvalues and right eigenvectors of the input matrix. Here's an example:
 ```
import numpy as np

A = np.array([[2, 1], [1, 2]])
eigvals, eigvecs = np.linalg.eig(A)

print("Eigenvalues:", eigvals)
print("Eigenvectors:\n", eigvecs)

```

This will output:

```
Eigenvalues: [3. 1.]
Eigenvectors:
 [[ 0.70710678 -0.70710678]
  [ 0.70710678  0.70710678]]

```

The eigvals array contains the eigenvalues of the matrix, and the columns of the eigvecs array contain the corresponding eigenvectors.

Note that the input matrix is not scaled in this function, and the resulting eigenvectors are not necessarily normalized. If you need normalized eigenvectors, you can normalize them using the numpy.linalg.norm() function.
